src/lib.rs

```diff
@@ -2002,6 +2010,15 @@ impl Connection {
         Ok(())
     }
 
+    /// Returns the stream's outgoing flow control capacity in bytes.
+    pub fn stream_capacity(&self, stream_id: u64) -> Result<usize> {
```

Comment:
< speculation>
It might be useful in the future to know the capacity of read/inbound streams. So perhaps we want to name this method `stream_send_capacity`.
< /speculation>

Reply:
I'd expect that to be called something other than "capacity" though, as that to me, mostly means what's the available space inside the buffer rather than what's the actual buffer size is (i.e. how much of the capacity is actually used). Though we can also always change this name later on as we are quite far from stabilizing the API :)

Reply:
yeah fair point, resolving this